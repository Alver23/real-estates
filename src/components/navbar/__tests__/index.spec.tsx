// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import Navbar from '../index';

describe('<Navbar />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Navbar />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
