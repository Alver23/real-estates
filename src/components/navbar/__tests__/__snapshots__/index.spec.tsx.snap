// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should save a snapshot of the component 1`] = `
<nav
  className="md:top-0 z-20 | w-full md:w-auto | bg-green-300 md:bg-white"
>
  <ul
    className="flex md:flex flex-col md:flex-row | md:space-x-6 text-white md:text-primary"
  >
    <li
      className="px-4 py-2 md:p-0 | border-b md:border-0"
      key="1"
    >
      <Link
        to="/"
      >
        Cundinamarca
      </Link>
    </li>
    <li
      className="px-4 py-2 md:p-0 | border-b md:border-0"
      key="2"
    >
      <Link
        to="/"
      >
        Antioquia
      </Link>
    </li>
    <li
      className="px-4 py-2 md:p-0 | border-b md:border-0"
      key="3"
    >
      <Link
        to="/"
      >
        Recursos
      </Link>
    </li>
    <li
      className="px-4 py-2 md:p-0"
    >
      <Link
        className="flex flex-row items-center relative"
        to="/"
      >
        <span
          className="pr-2"
        >
          Perfil
        </span>
        <MdKeyboardArrowDown
          className="hidden md:block"
          size={24}
        />
        <div
          className="absolute hidden md:block top-0.5 right-0"
        >
          <NotificationBar />
        </div>
      </Link>
    </li>
  </ul>
</nav>
`;
